src/bin/src/systems/tcp_listener_system.rs

```diff
@@ -29,12 +29,11 @@ impl TcpListenerSystem {
     async fn initiate_loop(state: GlobalState) -> Result<()> {
         let tcp_listener = &state.tcp_listener;
         info!("Server is listening on [{}]", tcp_listener.local_addr()?);
-
-
+        
         loop {
+            info!("Accepting connection");
```

Comment:
`info!` rahhh. Just use debug here, and when connection is established (when we have remote addr) then send `info` like "Accepted connection from {:?}" or something similar.

Reply:
sure